### PR TITLE
Porting of F-OH to API 11

### PR DESCRIPTION
**Pull request by frankplus**
Porting of F-OH to API 11 allowing F-OH to run on Oniro based on OpenHarmony-4.1-Release

### CODE REVIEW

File: AppScope/app.json5 [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-    "versionName": "1.3.5",
-    "icon": "$media:ONIRO",
-    "label": "$string:app_name",
-    "distributedNotificationEnabled": true
+    "bundleName": "com.example.appstore_sample",
+    "vendor": "example",
+    "versionCode": 1000000,
+    "versionName": "1.0.0",
+    "icon": "$media:app_icon",
+    "label": "$string:app_name"
```
Comment (frankplus) [inline AppScope/app.json5:8]: these changes doesn't seem to be necessary can you please revert?

File: AppScope/resources/base/element/string.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "string": [
     {
       "name": "app_name",
-      "value": "Oniro"
+      "value": "appStore_sample"
```
Comment (frankplus) [inline AppScope/resources/base/element/string.json]: this is also unnecessary

File: appStore_image/desktop.ini [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[LocalizedFileNames]
+problem_1.png=@problem_1.png,0
+problem_2.png=@problem_2.png,0
+problem_7.png=@problem_7.png,0
```
Comment (frankplus) [inline appStore_image/desktop.ini]: this file is an artifact from windows system and not needed

File: build-profile.json5 [hunk 4]
```diff
@@ -1,29 +1,34 @@
...
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
```
Comment (frankplus) [inline build-profile.json5:17]: you should not remove this copyright statement

File: entry/build-profile.json5 [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
```
Comment (frankplus) [inline entry/build-profile.json5:17]: you should not remove this copyright statement

File: entry/src/main/ets/abilitystage/MyAbilityStage.ts [hunk 6]
```diff
@@ -1,20 +1,3 @@
...
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
```
Comment (frankplus) [inline entry/src/main/ets/abilitystage/MyAbilityStage.ts:17]: you should not remove this copyright statement

File: entry/src/main/ets/components/AppListItem.ets [hunk 7]
```diff
@@ -1,22 +1,4 @@
...
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```
Comment (frankplus) [inline entry/src/main/ets/components/AppListItem.ets:16]: you should not remove this copyright statement

File: entry/src/main/ets/components/InfoRow.ets [hunk 8]
```diff
@@ -1,42 +1,38 @@
...
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```
Comment (frankplus) [inline entry/src/main/ets/components/InfoRow.ets]: you should not remove this copyright statement

File: entry/src/main/ets/components/Settings.ets [hunk 9]
```diff
@@ -1,24 +1,8 @@
...
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```
Comment (frankplus) [inline entry/src/main/ets/components/Settings.ets:16]: you should not remove this copyright statement

File: entry/src/main/ets/data/DataSource.ets [hunk 10]
```diff
@@ -1,28 +1,11 @@
...
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```
Comment (frankplus) [inline entry/src/main/ets/data/DataSource.ets:16]: you should not remove this copyright statement

File: entry/src/main/ets/model/AppInfo.ets [hunk 11]
```diff
@@ -1,41 +1,24 @@
...
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```
Comment (frankplus) [inline entry/src/main/ets/model/AppInfo.ets:16]: you should not remove this copyright statement

File: entry/src/main/ets/model/HomePageData.ets [hunk 12]
```diff
@@ -1,26 +1,10 @@
...
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```
Comment (frankplus) [inline entry/src/main/ets/model/HomePageData.ets:16]: you should not remove this copyright statement

File: entry/src/main/ets/pages/AboutApp.ets [hunk 13]
```diff
@@ -1,45 +1,33 @@
...
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```
Comment (frankplus) [inline entry/src/main/ets/pages/AboutApp.ets:16]: same here and the next ones

**Review by frankplus — COMMENTED** [review]
Hi, there are a lot of changes which also originates from the fact that you created a new project rather than updating the existing, which is ok especially when there are a lot of API changes. 

However can you please revert the changes that are not needed. Such as
- Revert the removal of copyright statements
- Revert the refactoring of project and application name
- Revert the removal of README.md, instead can you please move the notes that you wrote on API update,  to a directory `notes/` including the related pictures
- is the directory `entry/src/ohosTest/`  needed? if not can you delete it?

**Comment by imansmallapple** [comment]
> Porting of F-OH to API 11 allowing F-OH to run on Oniro based on OpenHarmony-4.1-Release

Hi I reverted all the unecessary changes and deleted ohos test file.

File: entry/src/main/resources/base/element/string.json [hunk 14]
```diff
@@ -48,6 +48,18 @@
       "name": "app",
       "value": "App"
     },
+    {
+      "name": "app_version",
+      "value": "Version"
+    },
+    {
+      "name": "oniro_plateform",
+      "value": "Oniro Plateform"
```
Comment (frankplus) [inline entry/src/main/resources/base/element/string.json]: Do you mean "platform"?

File: entry/src/main/resources/en_US/element/string.json [hunk 15]
```diff
@@ -48,6 +48,18 @@
...
+      "name": "app_version",
+      "value": "Version"
+    },
+    {
+      "name": "oniro_appStore",
+      "value": "Oniro App Store"
+    },
+    {
+      "name": "oniro_plateform",
+      "value": "Oniro Plateform"
```
Comment (frankplus) [inline entry/src/main/resources/en_US/element/string.json]: same here

**Review by frankplus — COMMENTED** [review]
apart from the comment, and squash the naming modification commit, for the rest looks good!

File: entry/src/main/resources/base/element/string.json (hunk 14 above)
Comment (imansmallapple) [reply entry/src/main/resources/base/element/string.json]: yes, I fixed this typo just now

**Comment by frankplus** [comment]
thanks, can you also squash the commits

**Comment by imansmallapple** [comment]
> thanks, can you also squash the commits

sure, squashed

**Pull request merged**